examples/interaction/js_callbacks/customjs_for_hover.py

```diff
@@ -34,22 +34,22 @@
 cr = p.scatter(x, y, color='olive', size=30, alpha=0.4, hover_color='olive', hover_alpha=1.0)
 
 # add a hover tool that sets the link data for a hovered circle
-code = """
-const links = %s
-const data = {'x0': [], 'y0': [], 'x1': [], 'y1': []}
+code = f"""
+const links = {links}
+const data = {{'x0': [], 'y0': [], 'x1': [], 'y1': []}}
 const indices = cb_data.index.indices
-for (let i = 0; i < indices.length; i++) {
+for (let i = 0; i < indices.length; i++) {{
     const start = indices[i]
-    for (let j = 0; j < links[start].length; j++) {
+    for (let j = 0; j < links[start].length; j++) {{
         const end = links[start][j]
         data['x0'].push(circle.data.x[start])
         data['y0'].push(circle.data.y[start])
         data['x1'].push(circle.data.x[end])
         data['y1'].push(circle.data.y[end])
```

Comment:
As long as we are already improving this file
```suggestion
        data.x0.push(circle.data.x[start])
        data.y0.push(circle.data.y[start])
        data.x1.push(circle.data.x[end])
        data.y1.push(circle.data.y[end])
```